frontend/static/scripts/project_index_scripts.js

```diff
@@ -25,11 +25,19 @@ function initOverview(projectsData, statusChoicesData) {
         }
     }
 
+    function filterSearch(searchTerm) {
+        return function (project) {
+            if (!searchTerm) return true;
+            var fields = project.title + " " + project.subtitle;
+            return fields.toLowerCase().indexOf(searchTerm.toLowerCase()) > -1;
+        }
+    }
+
     var v = new Vue({
         el: '#project_index_page',
         delimiters: ["[[", "]]"],
         methods: {
-            log: console.log,
+            log: console.log,           
```

Comment:
Dont add empty space at end of line
```suggestion
            log: console.log,
```